lib/matplotlib/image.py

```diff
@@ -728,19 +726,20 @@ def can_composite(self):
 
     def set_resample(self, v):
         """
-        Set whether or not image resampling is used.
+        Set whether image resampling is used.
 
         Parameters
         ----------
-        v : bool
+        v : bool or None
+            If None, use :rc:`image.resample`.
```

Comment:
It would be nice to give the default rcParams setting:
https://matplotlib.org/devdocs/devel/documenting_mpl.html?highlight=documenting#rcparams

Reply:
done